src/pkg/selectors/selectors.go

```diff
@@ -293,6 +293,27 @@ func (s Selector) PathCategories() (selectorPathCategories, error) {
 	return ro.PathCategories(), nil
 }
 
+// AllPathCategories returns the sets of include and filter path categories
+// across all scope sets.
+func (s Selector) AllPathCategories() ([]path.CategoryType, error) {
+	all, err := s.PathCategories()
+	if err != nil {
+		return nil, clues.Stack(err)
+	}
+
+	res := map[string]path.CategoryType{}
```

Comment:
Why use the string value here instead of the enum for the key?

Reply:
that's an excellent point. My brain died so I didn't think of it 🙃